src/pages/Patient/index.tsx

```diff
@@ -35,21 +36,30 @@ export const PatientsPage = () => {
           {t("createPatient")}
         </Button>
       </Box>
-      <PatientModal
```

Comment:
No need to  ti  check  isModalOpen &&  ...  Just render  element without  checking  .